test/projects/file-generate/README.md

```diff
@@ -0,0 +1,5 @@
+This is a dummy project used to test the `check-generate` make target for go operator (like in `openshift/golang-osd-operator`).
+
+#### Projects main points
+- This is an operator initialized with `operator-sdk v0.16` in order to have all the `make generate` commands working properly
```

Comment:
Not `v0.16`. Templated to allow substitution by the test case.

Reply:
In the [fup](https://github.com/openshift/boilerplate/pull/59) you left this in, but added explanation below it. I guess you're saying you set this test project up initially using 0.16? Not really sure how that's relevant. But meh.